client/jetpack-cloud/sections/settings/advanced-credentials/credentials-form/index.tsx

```diff
@@ -0,0 +1,238 @@
+				>
+					<option value="ssh">{ translate( 'SSH/SFTP' ) }</option>
+					<option value="ftp">{ translate( 'FTP' ) }</option>
+				</FormSelect>
+			</FormFieldset>
+
+			<div className="credentials-form__row">
+				<FormFieldset className="credentials-form__server-address">
+					<div className="credentials-form__support-info">
+						<FormLabel htmlFor="host-address">{ translate( 'Server Address' ) }</FormLabel>
+						{ hostInfo && hostInfo.serverAddress && (
+							<InfoPopover>
+								<Info info={ hostInfo.serverAddress } />
+							</InfoPopover>
+						) }
+					</div>
+					<FormTextInput
+						name="host"
+						id="host-address"
+						placeholder={ translate( 'YourGroovyDomain.com' ) }
+						// value={ get( form, 'host', '' ) }
+						// onChange={ handleFieldChange }
+						// disabled={ formIsSubmitting }
+						// isError={ !! formErrors.host }
+					/>
+					{ /* { formErrors.host && <FormInputValidation isError={ true } text={ formErrors.host } /> } */ }
+				</FormFieldset>
+
+				<FormFieldset className="credentials-form__port-number">
+					<div className="credentials-form__support-info">
+						<FormLabel htmlFor="server-port">{ translate( 'Port Number' ) }</FormLabel>
+						{ hostInfo && hostInfo.portNumber && (
+							<InfoPopover>
+								<Info info={ hostInfo.portNumber } />
+							</InfoPopover>
+						) }
+					</div>
+					<FormTextInput
+						name="port"
+						id="server-port"
+						placeholder="22"
+						// value={ get( form, 'port', '' ) }
+						// onChange={ handleFieldChange }
+						// disabled={ formIsSubmitting }
+						// isError={ !! formErrors.port }
+					/>
+					{ /* { formErrors.port && <FormInputValidation isError={ true } text={ formErrors.port } /> } */ }
+				</FormFieldset>
+			</div>
+
+			<FormFieldset className="credentials-form__path">
+				<div className="credentials-form__support-info">
+					<FormLabel htmlFor="wordpress-path">
+						{ translate( 'WordPress installation path' ) }
+					</FormLabel>
+					{ hostInfo && hostInfo.installationPath && (
+						<InfoPopover>
+							<Info info={ hostInfo.installationPath } />
+						</InfoPopover>
+					) }
+				</div>
+				<FormTextInput
+					name="path"
+					id="wordpress-path"
+					placeholder="/public_html/wordpress-site/"
+					// value={ get( form, 'path', '' ) }
+					// onChange={ handleFieldChange }
+					// disabled={ formIsSubmitting }
+					// isError={ !! formErrors.path }
+				/>
+
+				{ /* { formErrors.path && <FormInputValidation isError={ true } text={ formErrors.path } /> } */ }
+			</FormFieldset>
+
+			<SegmentedControl>
+				<SegmentedControl.Item
+					selected={ mode === Mode.Password }
+					onClick={ () => setMode( Mode.Password ) }
+				>
+					{ translate( 'Use password' ) }
+				</SegmentedControl.Item>
+				<SegmentedControl.Item
+					selected={ mode === Mode.PrivateKey }
+					onClick={ () => setMode( Mode.PrivateKey ) }
+				>
+					{ translate( 'Use private key' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **79** times:
`translate( 'Private Key' )` *ES Score: 6*